Apps/Examples/Examples/All Examples/SwitchStylesExample.swift

```diff
@@ -22,48 +68,31 @@ public class SwitchStylesExample: UIViewController, ExampleProtocol {
 
```

Comment:
> `mapView.mapboxMap.style.uri = .satelliteStreets`

I suppose we can drop this assignment as we already have one through the `MapInitOptions`.